contracts/AKAP.sol

```diff
@@ -26,6 +26,9 @@ contract AKAP is ERC721Full {
         bytes nodeBody;
     }
 
+    event NodeClaim(address indexed _sender, uint256 indexed _nodeId, uint _parentId, bytes _label);
+    event NodeChange(address indexed _sender, uint256 indexed _nodeId);
```

Comment:
Should we include some type of flag here to indicate what type of change caused the event?

Reply:
I initially had attribute and value parameters, to log the source of event dispatch. I think the attribute is enough?

NodeChange(address sender, uint nodeId, string attribute)

Reply:
Can we not use an enum from the change type, instead of string?

Reply:
Enum is more efficient and less error prone, I had a preference of showing the full function name in log. I thought enum might just show the ordinal value? I need to try it.

Reply:
> case

ERC721 triggers a transfer event (tokenId=nodeId), but probably good for consistency to emit contract 3rd case event.